src/core/webview/ClineProvider.ts

```diff
@@ -725,8 +726,28 @@ export class ClineProvider implements vscode.WebviewViewProvider {
 
 	// MCP
 
+	async getDocumentsPath(): Promise<string> {
+		if (process.platform === "win32") {
+			// If the user is running Win 7/Win Server 2008 r2+, we want to get the correct path to their Documents directory.
+			try {
+				const docsPath = execSync(
```

Comment:
please use execa to run commands (asynchronous and handles escaping nicely). you can see an example of this in integrations/notifications